stanford_news.info.yml

```diff
@@ -1,41 +1,43 @@
 name: 'Stanford News'
 type: module
 description: 'Stanford news items and views.'
-core: 8.x
+core_version_requirement: ^8.8 || ^9
 package: Stanford
-version: 8.x-1.x
+version: 8.x-1.0-dev
 dependencies:
-  - auto_entitylabel
-  - allowed_formats
-  - breakpoint
-  - components
-  - datetime
-  - ds
-  - ds_extras
-  - entity_reference_revisions
-  - field
-  - file
-  - image
-  - jumpstart_ui
-  - link
-  - menu_ui
-  - node
-  - paragraphs
-  - path
-  - pathauto
-  - stanford_media
-  - stanford_text_editor
-  - taxonomy
-  - text
-  - ui_patterns
-  - ui_patterns_layouts
-  - ui_patterns_library
-  - ui_patterns_views
-  - user
-  - views_block_filter_block
-  - views_infinite_scroll
-  - viewfield
-  - views_taxonomy_term_name_depth
+  - auto_entitylabel:auto_entitylabel
+  - allowed_formats:allowed_formats
+  - drupal:breakpoint
+  - components:components
+  - drupal:datetime
+  - ds:ds
+  - ds:ds_extras
+  - entity_reference_revisions:entity_reference_revisions
+  - drual:field
+  - drupal:file
+  - drupal:image
+  - jumpstart_ui:jumpstart_ui
+  - drupal:link
+  - drupal:menu_ui
+  - drupal:node
+  - paragraphs:paragraphs
+  - drupal:path
+  - pathauto:pathauto
+  - stanford_media:stanford_media
+  - stanford_text_editor:stanford_text_editor
+  - drupal:taxonomy
+  - drupal:text
+  - ui_patterns:ui_patterns
+  - ui_patterns:ui_patterns_layouts
+  - ui_patterns:ui_patterns_library
+  - ui_patterns:ui_patterns_views
+  - drupal:user
+  - views_block_filter_block:views_block_filter_block
+  - views_infinite_scroll:views_infinite_scroll
+  - viewfield:viewfield
+  - views_taxonomy_term_name_depth:views_taxonomy_term_name_depth
+  - page_manager:page_manager
+  - page_manager:page_manager_ui
```

Comment:
We can talk about this later.

Reply:
this is breaking install:

  - page_manager:page_manager
  - page_manager:page_manager_ui